crates/ironrdp-session/src/fast_path.rs

```diff
@@ -36,6 +37,7 @@ pub struct Processor {
     use_system_pointer: bool,
     mouse_pos_update: Option<(u16, u16)>,
     no_server_pointer: bool,
+    pointer_software_rendering: bool,
```

Comment:
Nitpick: "blending" sounds more accurate to me (over "rendering")

Reply:
I think naming is about right -- It not only controls how to draw the cursor to the framebuffer but also changes pointer messages generation a bit. I would rather keep more generic `rendering` wording